explainer.md

```diff
@@ -606,6 +640,10 @@ subset of what's required for cross-origin transitions :
     already has knowledge of the cross-origin policy on the new page. They can
     ensure not to reference cross-origin resources in the properties made live.
 
+    Note that painting cross-origin resources in the image generated by element()
+    is not a security concern since the pixel content will never be accessible to
+    the incoming page.
```

Comment:
I wonder if that is a Chrome implementation thing. I mean, there must be some reason we changed the behaviour of `<img>` on isolated pages.